### PR TITLE
Updated gi tests

### PR DESCRIPTION
**Pull request by wojciech-zagrajczuk**

### CODE REVIEW

File: gi_tests/generateTestSuit.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+dims = '2'
+dds = '1.0|2.0|3.0|4.0|5.0'
+dtas = '1.0|2.0|3.0|4.0|5.0'
+locs = 'true|false'
+lims = '2.0|20.0'
+comps = '1to1|gpr'
+epses = '0.1|0.05|0.01|0.005'
+
+
+pattern = re.compile("(dim)(" + dims + ")(dd)(" + dds + ")(dta)(" + dtas + ")(loc)(" + locs + ")(lim)(" + lims + ")(comp)(" + comps + ")(eps)(" + epses + ")")
```
Comment (kroczi) [inline gi_tests/generateTestSuit.py]: no need for parenthesis around dim, dd, etc.

File: gi_tests/generateTestSuit.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import sys
```
Comment (kroczi) [inline gi_tests/generateTestSuit.py]: generateTestSuit**e**.py

File: gi_tests/mainTest.cpp [hunk 3]
```diff
@@ -123,15 +141,12 @@ bool handleTestCase(ParseParameters* parametersParser, ParseData* dataParser, st
   }
 }
 
-int main()
+void handleTestSuit(ParseParameters* parametersParser, ParseData* dataParser, string path)
```
Comment (kroczi) [inline gi_tests/mainTest.cpp]: Suite

File: gi_tests/generateTestSuit.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        testCase = open(testCasePath, "w")
+        content = sys.argv[3] + "\n" + sys.argv[4] + "\n" + configPath + "\n"
+        if(len(sys.argv) == 6):
+            content = content + sys.argv[5] + "\n"
+        testCase.write(content)
+        testCase.close()
+
+        testSuitePaths.append(testCasePath)
+
+if not os.path.exists("./suits/test-suits/"):
```
Comment (kroczi) [inline gi_tests/generateTestSuit.py]: suites

File: gi_tests/parameters.h [hunk 5]
```diff
@@ -6,9 +6,12 @@ class Parameters
   double noDimentions;
```
Comment (kroczi) [inline gi_tests/parameters.h]: int noDimentions;

File: gi_tests/mainTest.cpp [hunk 6]
```diff
@@ -123,15 +141,12 @@ bool handleTestCase(ParseParameters* parametersParser, ParseData* dataParser, st
...
 {
-
-  ParseParameters* parametersParser = new ParseParameters();
-  ParseData* dataParser = new ParseData();
-
   ifstream testSuitFile;
```
Comment (patrykks) [inline gi_tests/mainTest.cpp]: testSuiteFile

**Pull request merged**